Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
```

Comment:
You are probably able to shave off some from the image size by using `python:alpine3.6`